src/main/java/com/github/mathematicalguy/moecraft/common/Village.java

```diff
@@ -0,0 +1,140 @@
+package com.github.mathematicalguy.moecraft.common;
+
+import com.github.mathematicalguy.moecraft.MoeCraftMod;
+import com.google.common.collect.ImmutableList;
+import com.mojang.datafixers.util.Pair;
+import it.unimi.dsi.fastutil.ints.Int2ObjectMap;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.merchant.villager.VillagerTrades;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+import net.minecraft.item.MerchantOffer;
+import net.minecraft.util.IItemProvider;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.gen.feature.jigsaw.JigsawManager;
+import net.minecraft.world.gen.feature.jigsaw.JigsawPattern;
+import net.minecraft.world.gen.feature.jigsaw.JigsawPiece;
+import net.minecraft.world.gen.feature.jigsaw.SingleJigsawPiece;
+import net.minecraft.world.gen.feature.structure.*;
+import net.minecraftforge.event.village.VillagerTradesEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Village {
+    public static final Random RAND = new Random();
+    public static final ResourceLocation CHEMIST = new ResourceLocation(MoeCraftMod.MOD_ID,"chemist");
+
+
+    //adding mr Conlon to the villager pool
+    public static void init() {
+        PlainsVillagePools.init();
+        SnowyVillagePools.init();
+        SavannaVillagePools.init();
+        DesertVillagePools.init();
+        TaigaVillagePools.init();
+
+
+        //will registers conlons house for each biome
+        for (String biome : new String[]{"plains", "snowy", "savanna", "desert", "taiga"})
+            addToPool(new ResourceLocation("village/" + biome + "/houses"), rl("village/houses/" + biome + "_chemist"), 1);
```

Comment:
As a matter of style if you're are going to do a loop or if statement on 2 lines you should use `{` `}` - it makes the code slightly easier to read, and can actually reduce some stupid mistackes